scripts/webhook-migration/updateWebhooks.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+#
+# Used to update GitHub webhooks.
+# See https://github.com/fabric8-services/fabric8-jenkins-proxy/issues/94
+
+# Make sure we have all the required environment variables injected
+[ -z "${TENTANT_URL}" ] && echo "TENTANT_URL needs to be set" && exit 1
+[ -z "${OPENSHIFT_URL}" ] && echo "OPENSHIFT_URL needs to be set" && exit 1
+[ -z "${OPENSHIFT_API_TOKEN}" ] && echo "OPENSHIFT_API_TOKEN needs to be set" && exit 1
+[ -z "${AUTH_TOKEN}" ] && echo "AUTH_TOKEN needs to be set" && exit 1
+
+# Ensure required tools are installed
+hash jq 2>/dev/null || { echo >&2 "'jq' is required."; exit 1; }
+hash curl 2>/dev/null || { echo >&2 "'curl' is required."; exit 1; }
+hash oc 2>/dev/null || { echo >&2 "'oc' is required."; exit 1; }
+hash token-generator 2>/dev/null || { echo >&2 "'token-generator' is required."; exit 1; }
+
+##################################
+# Processes the build configuration of
+# the OpenShift cluster and extracts GitHub
+# repo URLs which might need updating.
+# Globals:
+#   OPENSHIFT_URL
+#   OPENSHIFT_API_TOKEN
+#   NAME_SPACES
+#   REPOS
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+findGitHubRepos() {
+  oc login ${OPENSHIFT_URL} --token=${OPENSHIFT_API_TOKEN} > /dev/null
+
+  #local projects=$(oc projects -q)
+  local projects=( "hferents hferents-osiotest1 hferents-osiotest2" )
+  for project in ${projects}; do
+   oc project ${project} > /dev/null;
+   local tmp=($(oc get bc -o json | jq -r ' .items[] | select(.spec.source.git.uri) | .metadata.namespace + " " + .spec.source.git.uri'))
+   if [ ${#tmp[@]} -eq 0  ]; then
+    continue
+   fi
+   NAME_SPACES+=(${tmp[0]})
+   REPOS+=(${tmp[1]})
+  done
+}
+
+#######################################
+# Finds the UUIDs for the discovered namespaces
+# Globals:
+#   NAME_SPACES - Namespaces to iterate
+#   UUIDS       - Array which gets populated with UUIDs
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+findUUIDs() {
+  for i in "${!NAME_SPACES[@]}"; do
+    uuid=$(curl -sgSL -H "Authorization: Bearer ${AUTH_TOKEN}" ${TENTANT_URL}/api/tenants?namespace=${NAME_SPACES[$i]}\&master_url=${OPENSHIFT_URL} | jq -r .data[0].id)
+    [ -z "${uuid}" ] && echo "WARN: Unable to determine UUID for ${NAME_SPACES[$i]}" >&2
+    UUIDS[$i]=${uuid}
+  done
+}
+
+#######################################
+# For each UUID get an OSIO token
+# Globals:
+#   UUIDS      - User ids to iterate
+#   OSIO_TOKEN - Array which gets populated with OSIO tokens
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+getOsioTokens() {
+  for i in "${!UUIDS[@]}"; do
+    # TODO Issue #94 - Here we have to call token-generator.
```

Comment:
@alexeykazakov @vpavlin - here I need to convert a user's uuid to an OSIO token. Any chance we can change `token-generator` to take a single uuid as input and return the token as output? This way I could easily call `token-generator` in this loop iterating over uuids? 
Or is there a reason why currently token generation in batch?